build.xml

```diff
@@ -106,6 +106,7 @@
         <move file="${workspace}/source/composer.json" tofile="${packagepath}/docs/en/dev/composer.json" overwrite="true"/>
         <move file="${workspace}/source/composer.lock" tofile="${packagepath}/docs/en/dev/composer.lock" overwrite="true"/>
 
+        <exec command="${workspace}/source/build.php build:generate_vendor_doc --write-to '${packagepath}/docs/en/Composer Vendors.md'" checkreturn="true" passthru="true"/>
```

Comment:
do you really want a space in the file name? `Composer Vendors.md`

I think this will cause problems on some file systems.